scipy/stats/_stats_py.py

```diff
@@ -4236,6 +4236,15 @@ def pearsonr(x, y, *, alternative='two-sided'):
 def fisher_exact(table, alternative='two-sided'):
     """Perform a Fisher exact test on a 2x2 contingency table.
 
+    The null hypothesis is that the true odds ratio of the populations
+    underlying the observations is one, and the observations were sampled at
+    random from these populations. The statistic returned is the unconditional
```

Comment:
Hold off on merge, actually. I need to figure out how to work in something about the marginals of the table.